larray/inout/hdf.py

```diff
@@ -74,7 +74,7 @@ def read_hdf(filepath_or_buffer, key, fill_value=nan, na=nan, sort_rows=False, s
         _meta = attrs.metadata if 'metadata' in attrs else None
         if _type == 'Array':
             res = df_aslarray(pd_obj, sort_rows=sort_rows, sort_columns=sort_columns, fill_value=fill_value,
-                              parse_header=False)
+                              parse_header=False, dense=True)
```

Comment:
only use dense=True if _type has an explicit value, not if it was missing, otherwise we can't load a dataframe saved by pandas without larray